### PR TITLE
[HELP WANTED] Two word cmds warnings

### PR DESCRIPTION
**Pull request by anaisabel7**
Calling "default" as error message function when the first word of a several word command can be found in the prompt to Jarvis, but not the rest of required words. See issue #92

### CODE REVIEW

File: Jarvis/tests/test_default.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import unittest
+from Jarvis import Jarvis
+from mock import patch
+
+
+class DefaultTest(unittest.TestCase):
+
+    def setUp(self):
+        self.jarvis = Jarvis()
```
Comment (anaisabel7) [inline Jarvis/tests/test_default.py]: I could use some help understanding why this is a problem for **Codacy** given it uses the same syntax as as the test_parser.
The tests pass correctly too.
I cannot figure out how to solve the problem and make Codacy happy.

Comment (StefanosChaliasos) [inline Jarvis/tests/test_default.py]: It's ok don't worry.

**Comment by StefanosChaliasos** [comment]
can you provide an example of your functionality

**Comment by anaisabel7** [comment]
This is the behaviour previous to my changes (in the master branch):
in_master_branch
And this is how Jarvis behaves in this feature branch:
in_feature_branch

**Comment by anaisabel7** [comment]
@StefanosChaliasos is there anything else you want me to add / change / explain in here? :smiley_cat:

**Comment by StefanosChaliasos** [comment]
Thanks for your contributions

**Pull request merged**